### DISCUSSION TITLE
Configure celery_executor within job definition

### DISCUSSION THREAD

**Discussion by mmarionncern** [dagster-io/dagster#10859] [Q&A] [answered]
Dear dagster experts,

For remote processing purposes (workers run on remote machines), I need to configure the dagster celery_executor before defining my job and run dagit : 

from dagster_celery import celery_executor
@job(executor_def=celery_executor)
def parallel_job():
       ...

in order to select within the executor the celery backend and broker.

What would be the best way to setup the broker here?

Thank you very much by advance for your help!

Matthieu Marionneau

**Comment by alangenfeld** [comment, marked as answer]
https://docs.dagster.io/concepts/configuration/configured#configured-api

**Comment by mmarionncern** [comment]
Thanks a lot, I missed that in the doc
